README.md

```diff
@@ -80,16 +80,17 @@ Alternatively, you could pass the webhook as a configuration option.
-| `unsafeMaxLength`      | Maximum character length for the release notes before truncation. If maxLength is too high, messages can be dropped. [Read here](https://github.com/juliuscc/semantic-release-slack-bot/issues/26#issuecomment-569804359) for more information. | 2900          |
+| Option                 | Description                                                                                                                                                                                                                                     | Default                                                      |
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| `notifyOnSuccess`      | Determines if a succesfull release should trigger a slack message to be sent. If `false` this plugin does nothing on success.                                                                                                                   | false                                                        |
+| `notifyOnFail`         | Determines if a failed release should trigger a slack message to be sent. If `false` this plugin does nothing on fail.                                                                                                                          | false                                                        |
+| `onSuccessTemplate`    | Provides a template for the slack message object on success when `notifyOnSuccess` is `true`. See [templating](#templating).                                                                                                                    | undefined                                                    |
+| `onFailTemplate`       | Provides a template for the slack message object on fail when `notifyOnFail` is `true`. See [templating](#templating).                                                                                                                          | undefined                                                    |
+| `markdownReleaseNotes` | Pass release notes through markdown to slack formatter before rendering.                                                                                                                                                                        | false                                                        |
+| `slackWebhookEnVar`    | Environment variable to use to select the default Slack webhook value.                                                                                                                                                                          | SLACK_WEBHOOK                                                |
```

Comment:
I am a bit confused by the word *default*. Also, it sounds to me like it is an environment variable when it is a configuration for an environmental variable.

Maybe it could say something like:

> This decides what the environment variable for exporting the slack webhook is called.

I am not 100% satisfied with this sentence either but maybe something more in that direction.